### ISSUE TITLE
Predict Topt from protein sequence and OGT

### ISSUE THREAD

**Issue by yanshouyu** [Gangl2016/Optimal-growth-temperture-predictor#2]
Currently we can only get pre-calculated Topt from BLAST results. It would be very handy to predict the Topt directly from the protein sequence (or AA frequency) and OGT calculated in `predOGT`, using the RF regressor already trained but not provided in this package.

**Comment by mengqvist** [comment]
I agree that this would be handy. I will mark this as an enhancement and hope to get back to it when time allows.

**Comment by SheridanPO** [comment]
This would be very useful! Do you have any idea when you will get a chance to add this?

**Comment by MdUmar-tech** [comment]
When i use this command
tome predOGT --fasta test/proteomes/pyrococcus_horikoshii.fasta
I m getting error message
.
ImportError: cannot import joblib from 'sklearn.externals (C:\ProgramData\Anaconda3\lib\site-packages\sklearn\externals\joblib__init__.py)
Plese solve

**Comment by feiranl** [comment]
@MdUmar-tech You can change the import function in `predOGT.py`, instead of `from sklearn.externals import joblib` but use 
```
import sklearn.externals
import joblib
```

**Comment by MdUmar-tech** [comment]
Thank you
